iron-overlay-behavior.html

```diff
@@ -393,20 +392,8 @@
     },
 
     _onCaptureClick: function(event) {
-      // attempt to close asynchronously and prevent the close of a tap event is immediately heard
-      // on target. This is because in shadow dom due to event retargetting event.target is not
-      // useful.
-      if (!this.noCancelOnOutsideClick && (this._manager.currentOverlay() == this)) {
-        this._cancelJob = this.async(function() {
-          this.cancel();
-        }, 10);
-      }
-    },
-
-    _onClick: function(event) {
-      if (this._cancelJob) {
-        this.cancelAsync(this._cancelJob);
-        this._cancelJob = null;
+      if (event.path.indexOf(this) === -1) {
```

Comment:
What happened to the `this.noCancelOnOutsideClick` part? Also, I think `this._manager.currentOverlay() == this` is still relevant..